showcases/e2e/input/showcase-input.spec.ts

```diff
@@ -6,6 +6,7 @@ test.describe('DBInput', () => {
 	// TODO: Currently disable till we solved https://github.com/db-ui/mono/issues/2587
 	getDefaultScreenshotTest({
 		path: '03/input',
-		disableRules: ['color-contrast']
+		axeDisableRules: ['color-contrast'],
+		aCheckerDisableRules: ['label_ref_valid'] // TODO: There is an issue with datetime-local - https://github.com/IBMa/equal-access/issues/1910
```

Comment:
```suggestion
		axeDisableRules: ['color-contrast']
```

https://github.com/IBMa/equal-access/issues/1910 seems to have been fixed

Reply:
https://github.com/db-ui/mono/pull/2652#discussion_r1677886633

Reply:
opened https://github.com/db-ui/mono/issues/2860 for this